lib/main.js

```diff
@@ -1,9 +1,20 @@
 /*!
  * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
-
+import bedrock from 'bedrock';
+const {config} = bedrock;
 import './config.js';
-export {OAuth2Client} from './oauth2-client.js';
+import {getAuthClients} from './oauth2-client.js';
```

Comment:
It's generally a good idea to differentiate between authn and authz.

```suggestion
import {getAuthzClients} from './oauth2-client.js';
```